.gitignore

```diff
@@ -77,3 +77,9 @@ vagrant/puppet/modules/
 vagrant/puppet/.tmp
 .librarian
 .vagrant
+
+#Dirs created in dev
+OpenOversight/db_repository/
```

Comment:
Yep it looks like these changes won't require a migration [for the reason](http://docs.sqlalchemy.org/en/latest/orm/tutorial.html#orm-tutorial-relationship) @scuerda describes. (Btw we're actually using `sqlalchemy-migrate` for the database migrations here, we _can_ switch to something else like `flask-migrate` (which uses `alembic`) if we encounter issues)